server/src/services/domain/chart.ts

```diff
@@ -13,21 +13,72 @@ export const Transform = (situations: CountrySituationInfo[]): ChartModel => {
                     label: Status.Confirmed,
                     data: situations.map((x) => x.confirmed),
                     fill: false,
-                    borderColor: 'blue',
+                    borderColor: 'rgb(230, 165, 96)',
+                    backgroundColor: 'rgb(230, 165, 96)',
                 },
                 {
                     label: Status.Deaths,
                     data: situations.map((x) => x.deaths),
                     fill: false,
-                    borderColor: 'red',
+                    borderColor: 'rgb(222, 66, 91)',
+                    backgroundColor: 'rgb(222, 66, 91)',
                 },
                 {
                     label: Status.Recovered,
                     data: situations.map((x) => x.recovered),
                     fill: false,
-                    borderColor: 'green',
+                    borderColor: 'rgb(62, 148, 107)',
+                    backgroundColor: 'rgb(62, 148, 107)',
                 },
             ],
         },
+        options: {},
+    };
+};
+
+export const enrichWithTitle = (
+    model: ChartModel,
+    title: string
+): ChartModel => {
+    return {
+        ...model,
+        options: {
+            ...model.options,
+            title: {
+                display: true,
+                text: title,
+                fontColor: 'dark',
+                fontFamily: 'Helvetica',
+                fontSize: 28,
+            },
+        },
+    };
+};
+
+export const enrichWithType = (model: ChartModel, type: string): ChartModel => {
+    if (type === 'barStacked') {
```

Comment:
do they have typings?

Reply:
It seems tricky actually to add types here. Since options could vatry from time to time
I feel like *ChartModel* is typed enough.
This is the point when we should say stop cause we've got maximum from typescript 
### Types
And leave *ChartModel* as is!  👨‍🚀